subsys/bluetooth/host/Kconfig

```diff
@@ -12,13 +12,15 @@ config BT_LONG_WQ
 
 if BT_LONG_WQ
 config BT_LONG_WQ_STACK_SIZE
-	# Hidden: Long workqueue stack size. Should be derived from system
-	# requirements.
-	int
+	int "Long workqueue stack size."
```

Comment:
When a Kconfig changes to be prompt-able, should it be added to https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/releases/release-notes-4.2.rst#new-apis-and-options ?

Reply:
That's an interesting question. In principle it has been possible to modify the option until now as well, through Kconfig overlay files, whereas now it becomes settable through `prj.conf`.